src/commands/distribute/release.ts

```diff
@@ -268,7 +269,46 @@ export default class ReleaseBinaryCommand extends AppCommand {
         throw failure(ErrorCodes.Exception, `failed to set distribution group and release notes for release ${releaseId}`);
       }
     }
+  }
+
+  private async getDistributionGroup(client: AppCenterClient, releaseId: number, destination: string): Promise<models.DistributionGroupResponse> {
+    try {
+      const { result } = await clientRequest<models.DistributionGroupResponse>(async (cb) => {
+        client.distributionGroups.get(this.app.ownerName, this.app.appName, destination, cb);
+      });
+
+      return result;
+    } catch (error) {
+      if (error.statusCode === 404) {
+        throw failure(ErrorCodes.InvalidParameter, `Could not find group ${destination}`);
+      } else {
+        debug(`Failed to distribute the release - ${inspect(error)}`);
+        throw failure(ErrorCodes.Exception, `Could not add group ${destination} to release ${releaseId}`);
```

Comment:
I suggest changing the error to just express that there was a failure getting the distribution groups and remove the releaseId as an input as it's not needed here.